framework/base/Application.php

```diff
@@ -411,8 +414,6 @@ public function run()
      */
     abstract public function handleRequest($request);
 
-    private $_runtimePath;
-
```

Comment:
> Can we apply the fixer rule for everything that is not a private property?

Configured elements are positioned from the top of the class - if we don't want to change position of private properties, we won't be able to configure position of elements, that should be bellow private properties (constructor, destructor and magic methods).

I don't like current convention for private cache variables. Scattering them across the class makes it difficult to estimate what information a class instance holds. And I'm not sure what are the benefits of this approach.